lib/base/index.js

```diff
@@ -15,10 +16,12 @@ class BaseBot {
   devicename: void | string
   homeDir: string
   verbose: boolean
+  spawnedProcesses: ChildProcess[]
```

Comment:
I think @malgorithms mentioned somewhere in a previous PR that he wanted to do this by pid – we could also do that, but I don't see any harm in having a record of the `ChildProcess`es themselves.

Reply:
oh no agreed, this is probably much better.